bin/eslint.js

```diff
@@ -9,8 +9,14 @@
 
 "use strict";
 
-// to use V8's code cache to speed up instantiation time
-require("v8-compile-cache");
+/*
+ * If running in Node.js use V8's code cache to speed up instantiation time.
+ * This checks for presence of `Deno` on `globalThis`, because Deno in Node.js
+ * compatibility mode provides polyfill for all globals present in Node (eg. `process`).
```

Comment:
I think this comment could be more concise and to the point about why we don't want to run this in Deno, which to my understanding is because of the additional permission it requires.

```suggestion
 * Speed up instantiation time in Node.
 * If using Deno, skip this since it requires the additional `--allow-env` permission.
```

Reply:
Actually it's both - it requires env permissions and even if they are granted code cache is not available. I'll update the comment accordingly

Reply:
I updated the comment.